packages/docs/docs/docs/deploying.md

```diff
@@ -87,112 +91,73 @@ contract MyContract is Initializable {
 
-Now that everything has been setup, we are ready to deploy the project.
-To do so simply run:
+`Choose a contract:` **MyContract**
+`Select a network:` **local**
+`Do you want to run a function after creating the instance?` **Y**
+`Select a method:` **initialize**
 
-```console
-npx zos push
-```
+When it prompts you for initial variable values, you can give it any **int** and **string** such as:
 
-This command deploys `MyContract` to the specified network and prints its
-address. If your project added other
-contracts (using the `add` command) they would be deployed as well.
+`_x`: **42**
+`_s`: **hitchhiker**
 
-The `push` command also creates a `zos.dev-<network_id>.json` file with all the
-information about your project in this specific network, including the addresses of the
-deployed contract implementations in `contracts["MyContract"].address`.
-You can read more about this file format in the
-[configuration files](configuration.md#zos-network-json) section.
+The interactive command does the following:
 
-If a previous `zos.dev-<network_id>.json` was already created, the `push`
-command will just update said file with your last changes. For example, if you
-change the source of `MyContract` and call `zos push` again,
-ZeppelinOS will deploy a new version of `MyContract`
-and replace the entry in the json file's `contracts["MyContract"].address`.
+* Invokes `create` to compile your contract.
 
-An important thing
-to understand is that the contracts deployed by the `push` command are logic contracts and are not intended to be used directly, rather to be used by
-upgradeable instances, as we will see later in the
-[Upgrading your project](https://docs.zeppelinos.org/docs/upgrading.html) section.
+  > **Note**: The current version of ZeppelinOS relies on Truffle to compile contracts and
+  > determine the Solidity compiler version to use. If you want to use a different version,
+  > specify it in the Truffle configuration file. You can find more information in the
+  > Truffle documentation at:
+  > https://github.com/trufflesuite/truffle/releases/tag/v5.0.0-beta.0#specify-a-solcjs-version
 
-For example, you wouldn't interact with the "MyContract" that we just deployed. This contract is only intended for defining the behavior or logic of the contract. You can find the address of this logic contract in the `zos.dev-<network_id>.json` file, under `contracts.MyContract.address`.
+* Invokes `add` to add your contract to the project by registering it in the `zos.json` configuration file.
 
-So what _do_ you interact with? As we'll see in depth in the next guide, what you do is create an upgradeable instance for the logic contract, and interact with this instance instead. 
+* Invokes `push` to deploy the contract to the `local` network. If your project had contained other contracts, they would also be deployed.
```

Comment:
The reader doesn't know what push means in this context, or why is there a difference between push and create.

Reply:
We should also add another step which is actually creating the upgradeable instance (ie the proxy) and initializing it

Reply:
Re adding another step to create an upgradeable instance--is this not done in the "Upgrading" tutorial? Let's discuss it on Slack.

Reply:
Re this discussion of `push`, I think it's too much detail for a quickstart, but we don't have a better place to put it yet. I suggest we leave it here for now. I did remove the "Pushing contracts" heading because it seemed like it meant there was another step for the reader, and there isn't.